File: Course_01/C++ Language/Matrizes/cada_linha.cpp
#include <iostream>

using namespace std;

int main()
{
    int n, i, j;

    cout << "Qual a ordem da matriz? ";
    cin >> n;

    int mat[n][n], conteNeg(0), maior;

    for (i = 0; i < n; i++) {
        for (j = 0; j < n; j++) {
            cout << "Elemento [" << i << "," << j << "]: ";
            cin >> mat[i][j];
        }
    }

    cout << "MAIOR ELEMENTO DE CADA LINHA:" << endl;

    for (i = 0; i < n; i++) {
        maior = mat[i][0];
        for (j = 0; j < n; j++) {
            if (maior < mat[i][j]) {
                maior = mat[i][j];
            }
        }
        cout << maior << endl;
    }

    return 0;
}

File: Course_01/Java Language/Matrizes/cada_linha.java
package Matrizes;

import java.util.Locale;
import java.util.Scanner;

public class cada_linha {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
	
		int i, j, m;
		
		System.out.print("Qual a quantidade de linhas da matriz? ");
		m = sc.nextInt();
		
		int [][] matriz = new int [m][m];
		
		for (i = 0; i < m; i++) {
			for (j = 0; j < m; j++) {
				System.out.print("Elemento [" + i + "," + j + "]: ");
				matriz[i][j] = sc.nextInt();
			}
		}
		
		
		int maior;
		
		System.out.println("MAIOR ELEMENTO DE CADA LINHA: ");
		
		for (i = 0; i < m; i++) {
			maior = matriz[i][0];
			for (j = 0; j < m; j++) {
				if (maior < matriz[i][j]) {
					maior = matriz[i][j];
				}
			}
			System.out.println(maior);
		}
		
		sc.close();
	}
}
